src/OceanSeaIceModels/CrossRealmFluxes/stability_functions.jl

```diff
@@ -0,0 +1,127 @@
+import Base: -
+import Statistics
+
+#####
+##### Struct that represents a 3-tuple of momentum, heat, and water vapor
+#####
+
+struct SimilarityScales{U, T, Q}
+    momentum :: U
+    temperature :: T
+    water_vapor :: Q
+end
+
+-(a::SimilarityScales, b::SimilarityScales) = SimilarityScales(a.momentum - b.momentum, 
+                                                               a.temperature - b.temperature,
+                                                               a.water_vapor - b.water_vapor)
+
+Statistics.norm(a::SimilarityScales) = norm(a.momentum) + norm(a.temperature) + norm(a.water_vapor)
+
+function default_stability_functions(FT = Float64)
+
+    # Edson et al. (2013)
+    ψu = MomentumStabilityFunction(FT)
+    ψc = ScalarStabilityFunction(FT)
+
+    return SimilarityScales(ψu, ψc, ψc)
+end
+
+# Implementation of stability functions that follow Edson et al (2013)
+# We can swap them out easily if we define new types `NewStability` with a method `(f::NewStability)(ζ)`
+struct MomentumStabilityFunction{FT}
+    p₁ :: FT
+    p₂ :: FT
+    p₃ :: FT
+    p₄ :: FT
+end
+
+struct ScalarStabilityFunction{FT}
+    p₁ :: FT
+    p₂ :: FT
+    p₃ :: FT
+    p₄ :: FT
+    p₅ :: FT
+end
+
+function MomentumStabilityFunction(FT = Float64;
+                                   p₁ = convert(FT, 0.35),
+                                   p₂ = convert(FT, 0.7),
+                                   p₃ = convert(FT, 15),
+                                   p₄ = convert(FT, 10.15))
```

Comment:
I think it might be better to provide constructors with default parameters a different way like a function

```julia
function edson_2013_momentum_stability_function(FT, p....)
    return StabilityFunction(p...)
end
```

sketching it out